clients/da-vinci-client/src/main/java/com/linkedin/davinci/bootstrap/AbstractBootstrapper.java

```diff
@@ -0,0 +1,40 @@
+package com.linkedin.davinci.bootstrap;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.rocksdb.Options;
+import org.rocksdb.RocksDB;
+import org.rocksdb.RocksDBException;
+import org.rocksdb.RocksIterator;
+
+
+public abstract class AbstractBootstrapper implements RocksDbBootstrapper {
+  static {
+    RocksDB.loadLibrary();
+  }
+
+  private static final Logger LOGGER = LogManager.getLogger(AbstractBootstrapper.class);
+
+  protected void verifyBootstrap(String newDbDir) throws RocksDBException {
```

Comment:
This is method is a bit confusing. Why do we need to open the rocksdb to verify? If we are able to bootstrap from it, it means a SUCCESS.

Reply:
To verify that the DB can read the entities. Could there be any issues with the data that prevents rocksdb to read/open it? If not / if we don't need this, i can remove